protos/NUgus.proto

```diff
@@ -1347,286 +1346,35 @@ PROTO NUgus [
         # Accelerometer and Gyro are positioned 0.2m above torso base
         translation 0.000000 0.000000 0.2
         children [
-          # lis331dlh Accelerometer (See Section 1. of docs/Robot_Model_RoboCup_2021 for details on parameters)
+          #(See Section 1. of docs/Robot_Model_RoboCup_2021 for details on noise)
           # Axis: x forward, y left, z up
+          # Units are [m/s^2]
+          # The acelerometer measures the reaction forces over 3 axes in [m/s^2], thus,
+          # at rest the accelerometer should read +9.81 [m/s^2] in the z axis.
```

Comment:
Gravity should be down and z should be up!
```suggestion
          # at rest the accelerometer should read -9.81 [m/s^2] in the z axis.
```

Reply:
This is not how the accelerometer behaves in WeBots. It is based on the reaction force experienced by the accelerometer, thus, 9.81m/s^2 when z is up at rest.

Reply:
![image](https://user-images.githubusercontent.com/2320984/118954083-c40ca100-b9a0-11eb-942f-c020366641cc.png)

Reply:
https://cyberbotics.com/doc/reference/accelerometer

> An Accelerometer at rest with earth's gravity will indicate 1 g (9.81 m/s²) along the vertical axis. Note that the gravity can be specified in the gravity field in the WorldInfo node

I assume this means that if we set `gravity` to `-9.81` in WorldInfo we will get the behaviour we desire? Or will this just screw everything up?